packages/twenty-website/src/app/components/Breadcrumbs.tsx

```diff
@@ -19,7 +28,23 @@ const ExternalLinkItem = styled.a`
 `;
 
 const ActivePage = styled.span`
-  color: #818181;
+  color: ${Theme.text.color.secondary};
+  font-weight: ${Theme.font.weight.medium};
+`;
+
+const StyledSection = styled(React.Fragment)`
```

Comment:
Let's do a div instead? Saw an error "Warning: Invalid prop `className` supplied to `React.Fragment`. React.Fragment can only have `key` and `children` props." I'm guessing it's triggered by this?